core/src/main/java/com/graphhopper/routing/VirtualEdgeIterator.java

```diff
@@ -135,6 +132,12 @@ public EdgeIteratorState setName( String name )
     }
 
     @Override
+    public boolean getBoolean(int key, boolean _default, PMap kwargs)
```

Comment:
because, I used the PMap to pass the ```reverse``` setting of the routing algorithm further on. Thus it is contains a property not fixed for a weighting. So maybe right now, just use a boolean reverse instead of a PMap and refactor once we need more parameter?